nodes/config/ui_group.html

```diff
@@ -6,7 +6,11 @@
                 value: 'Group Name',
                 required: true
             },
-            page: { type: 'ui-page', required: true },
+            page: { type: 'ui-page', required: false },
```

Comment:
Any group will require a `page` on which to live, so we need to have this as `required: true`